tables/twRO/recvpackets.txt

```diff
@@ -1,1548 +1,1583 @@
+08C4 8 8 0
+08C5 6 6 0
+08C6 4 4 0
+08C0 -1 10 0
+08C1 2 2 0
+08C2 2 2 0
+08C7 -1 19 1
+08C8 34 34 1
+08C9 2 2 0
+08CA -1 8 0
+08CB -1 10 0
+097B -1 16 0
+08CC 109 109 0
+08CE 2 2 0
+08CF 10 10 0
+08D0 9 9 1
+08D1 7 7 1
+08D2 10 10 1
+08D3 10 10 0
+08D4 8 8 0
+08D5 -1 8 0
+08D6 6 6 0
+08D7 28 28 1
+08D8 27 27 1
+08D9 30 30 1
+08DA 26 26 1
+08DB 27 27 1
+08DC 26 26 1
+08DD 27 27 1
+08DE 27 27 1
+08DF 50 50 1
+08E0 51 51 1
+08E1 51 51 1
+08E2 27 27 0
+08E3 157 157 0
+08E4 6 6 0
+08FC 30 30 0
+08FD 6 6 0
+08FE -1 4 1
+08FF 24 24 1
+0900 -1 4 1
+0901 -1 4 1
+0902 -1 4 1
+0903 -1 4 1
+0904 -1 4 1
+0905 -1 4 1
+0906 -1 45 1
+0907 5 5 0
+0908 5 5 1
+090A 26 26 1
+090D -1 10 1
+090E 2 2 0
+0910 10 10 1
+0911 30 30 1
+0912 10 10 1
+0913 30 30 1
+0914 -1 84 1
+090F -1 77 1
+0915 -1 78 1
+0916 26 26 1
+096B 4 4 1
+096C 6 6 1
+096D -1 4 1
+096E -1 4 1
+096F 7 7 1
+0971 6 6 1
+0972 -1 9 1
+0973 7 7 1
+0974 2 2 1
+0975 -1 28 1
+0976 -1 28 1
+0977 14 14 1
+0978 6 6 0
+0979 50 50 0
+097A -1 8 1
+097C 4 4 0
+097D 288 288 1
+097E 12 12 1
+097F -1 8 1
+0980 7 7 1
+0981 -1 12 0
+0982 7 7 0
+0983 29 29 1
+0984 28 28 1
+0985 -1 4 1
+0986 10 10 0
+0987 -1 41 0
+0988 6 6 0
+0989 2 2 0
+098A -1 74 0
+098D -1 4 0
+098E -1 28 0
+098B 2 2 0
+098C 4 4 0
+098F -1 8 0
+0990 41 41 1
+0991 -1 4 1
+0992 -1 4 1
+0993 -1 4 1
+0994 -1 4 1
+0995 -1 28 1
+0996 -1 28 1
+0997 -1 45 1
+0998 8 8 1
+0999 11 11 1
+099A 9 9 1
+099B 8 8 1
+099C 6 6 0
+099D -1 4 0
+099E 12 12 0
+099F -1 22 1
+09A0 6 6 0
+09A1 2 2 0
+09A2 6 6 0
+09A3 -1 8 0
+09A4 18 18 0
+09A5 7 7 0
+09AB 6 6 0
+09A6 12 12 0
+09A7 10 10 0
+09A8 16 16 0
+09A9 10 10 0
+09AA 16 16 0
+09AC -1 8 0
+09AD 12 12 0
+09AE 19 19 0
+09AF 4 4 0
+09B0 10 10 0
+09B1 4 4 0
+09B2 10 10 0
+09B3 6 6 0
+09B4 6 6 0
+09B5 2 2 0
+09B6 6 6 0
+09B7 4 4 0
+09B8 6 6 0
+09B9 4 4 0
+09BA 2 2 0
+09BB 6 6 0
+09BC 6 6 0
+09BD 2 2 0
+09BE 2 2 0
+09BF 4 4 0
+09C1 10 10 0
+09C2 -1 4 0
+09C3 10 10 0
+09C4 10 10 0
+09C5 1042 1042 0
+09C6 -1 16 0
+09C7 18 18 0
+09C8 -1 13 0
+09C9 -1 4 0
+09CA -1 23 1
+09CB 17 17 1
+09CC -1 4 0
+09CD 8 8 1
+09CE 102 102 1
+09CF -1 4 0
+09D0 -1 4 0
+09D1 14 14 1
+09D2 -1 28 0
+09D3 -1 28 0
+09D4 2 2 0
+09D5 -1 4 1
+09D6 -1 4 1
+09D8 2 2 1
+09D7 -1 5 1
+09D9 4 4 0
+09DA -1 8 0
+09DB -1 88 1
+09DC -1 81 1
+09DD -1 82 1
+09DE -1 33 1
+09DF 7 7 1
+09E0 -1 40 0
+09E1 8 8 1
+09E2 8 8 1
+09E3 8 8 1
+09E4 8 8 1
+09E5 18 18 1
+09E6 24 24 1
+09E7 3 3 1
+09E8 11 11 1
+09E9 2 2 1
+09EE 11 11 1
+09EF 11 11 1
+09F0 -1 7 1
+0A7D -1 7 1
+09F5 11 11 1
+09F6 11 11 1
+09EA 11 11 1
+09EB -1 24 1
+09F7 77 77 1
+09EC -1 64 1
+09ED 3 3 1
+09F3 11 11 1
+09F4 12 12 1
+09F1 11 11 1
+09F2 12 12 1
+0A04 6 6 1
+0A05 63 63 1
+0A06 6 6 1
+0A07 9 9 1
+0A03 2 2 1
+0A08 26 26 1
+0A12 27 27 1
+0A13 26 26 1
+0A14 10 10 1
+0A51 34 34 1
+0A32 2 2 0
+09F8 -1 8 1
+09F9 143 143 1
+09FA -1 6 1
+09FB -1 8 0
+09FC 6 6 1
+09FD -1 90 1
+09FE -1 83 1
+09FF -1 84 1
+0A00 269 269 1
+0A01 3 3 0
+0A02 4 4 1
+0A09 55 55 1
+0A0A 57 57 1
+0A0B 57 57 1
+0A0C 66 66 1
+8B00000A -1 4 1

```

Comment:
need to check this packet

Reply:
fixed

Reply:
@alisonrag do we only need to check the last packet or all three?